### ISSUE TITLE
[EN][Spell][Mage]

### ISSUE THREAD

**Issue by Fearyourself** [Atlantiss/BugTracker#6139]
When casting Polymorph it doesn't take the caster neither the target in combat.

http://www.wowhead.com/spell=118/polymorph

Same with Hex.

**Comment by blinkmaster** [comment]
dunno about hex but with poly this is intended cata mechanic

**Comment by Fearyourself** [comment]
No. No it's not. Why would a spell cast on an enemy not bring them in combat? All spells/abilities brings the target in combat except for Sap since it can only used when the target is not in combat.

Get your facts straight.

**Comment by Kejmil** [comment]
it was changed (at least for pve) in one of patches in cata, have you any proofs that it should give combat in pvp?

**Comment by Fearyourself** [comment]
It was changed in pve yes, but not pvp wise. It doesn't matter what spell you cast, you'll bring the target in combat no matter what. Besides there's a reason spells like fear brings players in combat. Correct me if I'm wrong.

**Comment by blinkmaster** [comment]
" Ya this has been like this for a while, so I guess it is working as intended. I used to ice trip all the ally on my PVE server by the Tiller's farm and never get flagged just as an annoyance to them. This was changed in the game in Cata when they made it so CC no longer caused threat, and also, CC no longer caused flagging. It might have originally been an oversight in the PVP world that was never really fixed, but considering it's been years now and they haven't, I think it's just working as intended."

"Been like this for a long time. It was put in to prevent pulling mobs in PvE when CC before a pull. All CC but fear is like that."

http://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-exploits/497962-pvp-polymorph-does-not-put-you-combat.html

http://us.battle.net/wow/en/forum/topic/2743689292

so get ur facts straught yourself next time.
